dashboards/scripts/get_partner_field_ids.py

```
@@ -0,0 +1,78 @@
+"""
+import json
+import sys
+import urllib.request
+
+
+def get_session(metabase_url, username, password):
+    req = urllib.request.Request(
+        f"{metabase_url}/api/session",
+        data=json.dumps({"username": username, "password": password}).encode(),
+        headers={"Content-Type": "application/json"},
+    )
```

Ruff (0.15.7)

[error] Audit URL open for permitted schemes. Allowing use of `file:` or custom schemes is often unexpected.

(S310)